changelog/_unreleased/2021-04-01-resolve-symfony-5-1-deprecations.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Resolve Symfony 5.1 deprecations
+issue: NEXT-14586
+---
+# Core
+* Resolved Symfony 5.1 routing deprecation, by explicitly setting `framework.router.utf8` to `true`.
```

Comment:
The changelog validator allows [certain keywords](https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-Implement-New-Changelog.md#changelog-entries) only to encourage a more uniform style of writing.

```suggestion
* Changed `Core/Framework/Resources/config/packages/dev/routing.yaml` and `Core/Framework/Resources/config/packages/routing.yaml` to address the Symfony 5.1 routing deprecation, by explicitly setting `framework.router.utf8` to `true`.
```